<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_material_layer</class>
 <widget class="QWidget" name="form_material_layer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Material Layer</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit_name"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QComboBox" name="comboBox_type">
     <item>
      <property name="text">
       <string>Lambert Difusion</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Phong Specular</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Specular Reflection</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Specular Transmission</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dielectric Reflection</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Conductor Reflection</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fresnel Transmission</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Scale</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_scale">
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
